### PR TITLE
Adding nodejs express documentation

### PR DESCRIPTION
**Pull request by amitkanfer**

### CODE REVIEW

File: docs/website/data/integrations.yaml
Comment (anderseknert) [inline docs/website/data/integrations.yaml:442]: I guess this was done by mistake?

**Review by anderseknert — COMMENTED** [review]
See comment about possible mistake. Except that this looks a great addition!

File: docs/website/data/integrations.yaml
Comment (amitkanfer) [reply docs/website/data/integrations.yaml]: Ooopss.. No idea how this happened!

**Review by anderseknert — COMMENTED** [review]
Looks fine now - just squash the commits and this should be good to go 👍

**Review by anderseknert — DISMISSED** [review]

**Review by srenatus — DISMISSED** [review]

**Comment by amitkanfer** [comment]
@anderseknert can we merge?

**Comment by anderseknert** [comment]
@amitkanfer yes, but you'll need to re-squash your commits so that there's only one of them left to merge. Once done I'll merge right away.

**Comment by anderseknert** [comment]
Also note that there was just another merge to the integrations.md just now so you'll want to rebase on top of master first.

**Comment by srenatus** [comment]
🔍 There's a typo in your commit message: `ndoejs` -> `nodejs` 😅

**Review by anderseknert — APPROVED** [review]

**Pull request merged**
